terra/templates/terra/employee_type_list.html

```diff
@@ -0,0 +1,136 @@
+{% extends "terra/base.html" %}
+
+{% load terra_extras %}
+
+{% block title %}Employee Type Report{% endblock %}
+
+{% block supplemental_css %}
+<style>
+    body {
+        padding-top: 6.5rem;
+
+}
+</style>
+{% endblock %}
+
+{% block body %}
+    <div class="row">
+        <div class="col">
+            <h1>Employee Type Report</h1>
+            <h2>Fiscal Year Placeholder</h2>
```

Comment:
@aprigge This should show the actual fiscal year

Reply:
Thanks, @akohler. I just made that change.